.travis.yml

```diff
@@ -17,12 +17,13 @@ cache:
     - $HOME/.ghc
     - $HOME/.cabal
     - $TRAVIS_BUILD_DIR/.cache
+    - $TRAVIS_BUILD_DIR/node_modules
 before_install:
   - $SHELL --version 2> /dev/null || dpkg -s $SHELL 2> /dev/null || which $SHELL
   - curl --version
   - wget --version
   - if [ -n "${SHELLCHECK-}" ]; then cabal update && cabal install transformers-0.4.3.0 ShellCheck && shellcheck --version ; fi
-  - if [ -z "${SHELLCHECK-}" ]; then nvm install node && npm install && npm ls urchin doctoc; fi
+  - if [ -z "${SHELLCHECK-}" ]; then nvm install node && npm install && npm prune && npm ls urchin doctoc; fi
```

Comment:
I agree, that's the place what it already been for a while, I can send another commit to move both `npm` and `cabel` into `install` section, what do you think?